### ISSUE TITLE
Can't get psycopg2 to run on OSX(10.11) el capitan 

### ISSUE THREAD

**Issue by lucas-aragno** [psycopg/psycopg2#481]
`easy_install` and `pip` are throwing this error:

``` bash
ld: warning: ignoring file /usr/local/lib/libpq.dylib, file was built for x86_64 which is not the architecture being linked (i386): /usr/local/lib/libpq.dylib
ld: in '/usr/local/lib/libunwind.dylib', file was built for x86_64 which is not the architecture being linked (i386): /usr/local/lib/libunwind.dylib for architecture i386
clang: error: linker command failed with exit code 1 (use -v to see invocation)
error: Setup script exited with error: command 'cc' failed with exit status 1
```

so I'm not that familiar with C I am missing any C library? should I change any config/syslink?

**Comment by Microserf** [comment]
As a first step, open XCode and see if there's an updated Terms of Service screen you need to Agree to. When Apple updates their ToS, they make it so that the `cc` and `gcc` compilers stop working until you've acknowledged and agreed to the updated terms.

Once you've done that, try again, and see if it now works.

**Comment by lucas-aragno** [comment]
I tried that but XCode doesn't have any new Terms of Serivces for me to agree on, Just ran `sudo xcodebuild -license` and agreed but still have the same issue :(

**Comment by dvarrazzo** [comment]
I cannot fix this specific problem. It will be fixed releasing Mac OSX wheels (bug #479). If you can provide help with them you are very welcome.

**Comment by dvarrazzo** [comment]
I have uploaded some test OSX wheels package on testpypi. Could you please check if they work?

    pip install -i https://testpypi.python.org/pypi psycopg2==2.7b1

Thank you very much.
